beta-src/src/utils/state/gameApiSlice/extraReducers/fetchGameOverview/fulfilled.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-param-reassign */
+export default function fetchGameOverviewFulfilled(state, action): void {
+  state.apiStatus = "succeeded";
+  state.overview = action.payload;
+  state.activity.makeNewCall = false;
+  const { processTime, members } = action.payload;
+  if (processTime) {
+    const membersPlaying = members.filter(({ status }) => status === "Playing");
+    // eslint-disable-next-line no-bitwise
+    const now = (Date.now() / 1000) | 0;
```

Comment:
why | instead of ||

Reply:
This is truncating the decimal, similar to Math.trunc but slightly more performant.